script.js

```diff
@@ -0,0 +1,52 @@
+/* Abstraction of game board for bonus features (i.e. checking if player won the game.)
+var board = [[null, null, null],[null, null, null],[null, null, null]];
+console.log(board);
+*/
+
+var turnCounter = 0;
+
+// Select the board, as the parent element of the cells that will be changed, and add event listener
+var gameboard = document.getElementById("board");
+gameboard.addEventListener("click", changeSymbol, false);
+
+// Select the reset button, add event listener
+var resetButton = document.getElementsByClassName("reset")[0];
+resetButton.addEventListener("click", resetBoard, false);
+
+function changeSymbol(e) {
+  // Console.log the DOM element being captured (for debugging)
+  //console.log(e);
+
+  // Switch the element's symbol if it's not the parent
+  if (e.target !== e.currentTarget){
+
```

Comment:
i'm not entirely sure what currentTarget is here, I think that it doesn't exist normally, so i imagine that it will always return true? because currentTarget will always be undefined.